app/views/comments/update.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#<%= dom_id(@comment.photo) %>_<%= dom_id(@comment) %>_form').replaceWith('<%= j render @comment %>');
```

Comment:
This works! Consider putting this in a variable next time to make your code more legible.